mkposters/mkposter.py

```diff
@@ -16,10 +18,39 @@ def md_to_html(md):
     return markdown.markdown(
         md,
         extensions=["admonition", "pymdownx.superfences", "smarty"],
+        extension_configs={
+            "pymdownx.superfences": {
+                "custom_fences": [
+                    {"name": "mermaid", "class": "mermaid", "format": fence_div_format}
+                ]
+            }
+        },
     )
 
 
-def mkposter(datadir):
+def mkposter(
+    datadir: str,
+    code_style: Optional[str] = None,
+    mermaid: Optional[bool] = False,
```

Comment:
I don't think is `Optional`, just `bool`.

Reply:
The goal with the optional type and default of None was to preserve backwards compatibility. Or do you suggest we move forward with these as required args, with no default values, and then bump the minor version?

Reply:
Sorry, I'm talking about the `mermaid` argument here, not the `code_style` argument. I just mean the existing behaviour is already `mermaid=False`, and passing `mermaid=None` does nothing.